command: /usr/bin/python /home/arigo/pypysrc/pypy/tool/alarm.py 900 /home/arigo/pypysrc/pypy/interpreter/py.py --oldstyle /home/arigo/pypysrc/lib-python-2.3.4/test/test_asynchat.py
run by: arigo@tismerysoft.de
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 05:11:20 2005
pypy-revision: 11518
OUTPUT TEST
see output in: /home/arigo/pypysrc/lib-python-2.3.4/test/result/test_asynchat.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
Traceback (application-level):
  File "/home/arigo/pypysrc/lib-python-2.3.4/test/test_asynchat.py", line 3 in ?
import thread # If this fails, we can't test this module
ImportError: thread
========================== closed ==========================
execution time: 0.944401025772 seconds
exit status: 1
